src/main/java/org/jetbrains/id/names/suggesting/contributors/NGramVariableNamesContributor.java

```diff
@@ -59,6 +75,22 @@ private static boolean isSupported(@NotNull PsiNameIdentifierOwner identifierOwn
                 .stream();
     }
 
+    public static Boolean caretInsideVariable(@NotNull PsiVariable variable) {
+//        TODO: Очень большой костыль. Если убрать nonBlocking, то intellij ругается, не может читать спокойно.
+        AtomicReference<Editor> editor = new AtomicReference<>(null);
+        ReadAction.nonBlocking(() -> {
```

Comment:
This code duplicates part of `utils.PsiUtils#caretInsideMethodBlock`. Should we extract the common part?